PCLConfig.cmake.in

```diff
@@ -239,7 +239,7 @@ macro(find_VTK)
       set(VTK_DIR "${PCL_ROOT}/3rdParty/VTK/lib/vtk-@VTK_MAJOR_VERSION@.@VTK_MINOR_VERSION@" CACHE PATH "The directory containing VTKConfig.cmake")
     endif()
   elseif(NOT VTK_DIR AND NOT ANDROID)
-    set(VTK_DIR "@VTK_DIR@" CACHE PATH "The directory containing VTKConfig.cmake")
+    set(VTK_DIR "@VTK_DIR@" CACHE PATH "The directory containing VTKConfig.cmake" FORCE)
```

Comment:
Sorry for late replay, but you sure its here you want the Force?
This is where VTK_DIR is not set to any value, I guess it should be able to update it?
The issue you link to, uses the All-In-One installer, which are line 237 and 239 - which probably make sense, since the All-in-one installer is build with that specific version.

Reply:
I believe so. 
The intention to add it in this file is that it would end up generating the `PCLConfig.cmake` mentioned in https://github.com/PointCloudLibrary/pcl/issues/4926
With `FORCE`, it allows the user to override the cache even if a value was already defined for `VTK_DIR`, which it does because it was added by the installer.